test/Serialization/Fields.jl

```diff
@@ -0,0 +1,88 @@
+@testset "Fields" begin
+    mktempdir() do path
+        @testset "UV Polynomial Over Simple Extension" begin
+            R, x = PolynomialRing(QQ, "x")
+            p = x^2 + 3//4
+            K, a = NumberField(p)
+            Ky, y = K["y"]
+            q = a * y^2 + y
+            filename = joinpath(path, "polynomial_se.uv")
+            save(q, filename)
+            loaded = load(filename)
+            L = base_ring(parent(loaded))
+            h = hom(K, L, gen(L))
+            @test [h(c) for c in coefficients(q)] == collect(coefficients(loaded))
```

Comment:
Try this instead? If it works, I recommend using it in the other tests, too
```suggestion
            @test map_coefficients(h, q) == loaded)
```

Reply:
doesn't seem to work
```
julia> @test map_coefficients(h, q) == loaded
Test Failed at REPL[28]:1
  Expression: map_coefficients(h, q) == loaded
   Evaluated: _a*y^2 + y == _a*y^2 + y
ERROR: There was an error during testing
```

Reply:
Parents are wrong. The `map_coefficients(h, q)` creates a new distinct parent.

Reply:
Ouch. Is there a variant of `map_coefficients` that takes the target ring as argument? If not, should there be one, or what else is the "best" alternative?